### PR TITLE
Issue #2873: Fix broken installation requirements page

### LINKED ISSUE #2873: [UX] Installation requirements page looks broken
## Describe your issue or idea

When using the web installer, the requirements page looks broken because the table is larger than the available space. Probably because this page use a `body.maintenance-page` class. As far as I see, there is no specific "install" classes.

### Steps to reproduce (if reporting a bug)

Try to install a Backdrop with bad `files` folder permission

### Actual behavior (if reporting a bug)

image

### Expected behavior (if reporting a bug)

**Solution 1**: Wrap the text of the requirements labels
image

**Solution 2**: Make the requirements page wider
image

**Solution 3**: Move the requirements labels above the respective status/details text
image

### PR DESCRIPTION
**Pull request by klonos**
Fixes https://github.com/backdrop/backdrop-issues/issues/2873

### CODE REVIEW

File: core/modules/system/system.theme.inc [hunk 1]
```diff
@@ -341,7 +347,7 @@ function theme_status_report($variables) {
   $system_status_report_table = array(
     'header' => array(),
     'rows' => array(),
-    'attributes' => array('class' => 'system-status-report'),
+    'attributes' => array('class' => array('system-status-report', 'phase-' . $phase)),
```
Comment (jenlampton) [inline core/modules/system/system.theme.inc:350]: I love this.

**Comment by klonos** [comment]
I've created a follow-up PR for the cspell nagging here: https://github.com/backdrop/backdrop/pull/4544

**Comment by klonos** [comment]
The cspell nagging is to be addressed in a separate PR here: https://github.com/backdrop/backdrop/pull/4544

The php 5.6 failure seems random, so kicking things again...

**Comment by klonos** [comment]
php 5.6 failed again - different random failure this time 🤷🏼

File: core/modules/system/system.theme.inc [hunk 2]
```diff
@@ -285,11 +285,17 @@ function theme_system_modules_uninstall($variables) {
  *       - REQUIREMENT_OK: The requirement is satisfied.
  *       - REQUIREMENT_WARNING: The requirement failed with a warning.
  *       - REQUIREMENT_ERROR: The requirement failed with an error.
+ *   - phase: The current phase. Possible values are one of:
+ *     - install.
+ *     - update.
+ *     - runtime (the default, if nothing else specified).
```
Comment (quicksketch) [inline core/modules/system/system.theme.inc]: Maybe elaborate on what each of these mean?

```suggestion
 *     - install: When called from install.php.
 *     - update: When called from update.php.
 *     - runtime: Showing the normal status report page, or no phase specified.
```

**Comment by quicksketch** [comment]
Great, merged into 1.x and 1.26.x. Thanks!

**Pull request merged**